dev/com.ibm.ws.microprofile.config.1.2_fat/build.gradle

```diff
@@ -10,5 +10,5 @@
  *******************************************************************************/
 
 dependencies {
-  requiredLibs 'commons-logging:commons-logging:1.1.3', 'commons-codec:commons-codec:1.6'
+  requiredLibs 'commons-logging:commons-logging:1.1.3', project(':com.ibm.ws.microprofile.config_fatshared'), project(':com.ibm.websphere.org.eclipse.microprofile.config.1.2.1')
```

Comment:
I assume this leads to these projects being packaged in the autoFVT.zip?

Couldn't find any information in the wiki, we should probably document this.

Reply:
yes that is the effect but i don't know how it actually works, i copied it from somewhere else